src/pages/signin.tsx

```diff
@@ -35,11 +37,50 @@ export default function Signin() {
 
   const [serverErrorState, setServerError] = useState<string | null>(null);
 
-  const onSubmitHandler: SubmitHandler<FormValues> = async () => {
+  const router = useRouter();
+
+  const onSubmitHandler: SubmitHandler<FormValues> = async (formData) => {
     try {
       // @TODO Implement submit
       // await onSubmit(formData.email, formData.password);
 
+      // Local Test
+      // axios({
+      //   method: 'post',
+      //   url: '/api/v1/auth/signin',
+      //   baseURL: 'http://localhost:3000',
+      //   data: {
+      //     email: formData.email,
+      //     password: formData.password,
+      //   },
+      // })
+      //   .then((response) => {
+      //     const resToken = response.data.access_token;
+      //     if (resToken.length > 0) {
+      //       router.push('/');
+      //     }
+      //   })
+      //   .catch((error) => {
+      //     document.getElementById('error_message').textContent =
+      //       'User does not exist or the wrong password was used!';
+      //   });
+
+      axios
+        .post('/api/v1/auth/signin', {
+          email: formData.email,
+          password: formData.password,
+        })
+        .then(function (response) {
+          const resToken = response.data.access_token;
+          if (resToken.length > 0) {
+            router.push('/');
+          }
+        })
+        .catch(function (error) {
+          document.getElementById('error_message').textContent =
+            'User does not exist or the wrong password was used!';
+        });
+
       reset(DEFAULT_FORM_VALUES);
       setServerError(null);
     } catch (error) {
```

Comment:
```suggestion
    } catch (error) {
        setServerError(error.response.data.message);
    }
```
All you have to do is `setServerError` to the response data message you are reciving from the server. When you get to refactor the code to use `async`, `await`